### PR TITLE
ci: fix cypress

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

Fix `cypress` setup for `racoon/onboarding`.

All of the changes were added as a result of running a migration tool `cypress` has made available on its new version.

## Justify why they are needed

`cypress` got updated 2 weeks ago to a major version - v10. As a result some migration were necessary.

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #753** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/753" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/753?utm_source=stack-comment).

File: apps/onboarding/cypress.config.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+export default defineConfig({
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      return require('./cypress/plugins/index.ts')(on, config)
+    },
+    baseUrl: 'http://localhost:3001',
```
Comment (KarnellSchultz) [inline apps/onboarding/cypress.config.ts:11]: Guess everything is going `ts` now 🚀

**Review by KarnellSchultz — APPROVED** [review]

File: apps/onboarding/cypress.config.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
```
Comment (robinandeer) [inline apps/onboarding/cypress.config.ts:6]: Should we do this already?

**Review by robinandeer — APPROVED** [review]
Nice!

File: apps/onboarding/cypress.config.ts (hunk 2 above)
Comment (guilhermespopolin) [reply apps/onboarding/cypress.config.ts:6]: Will do

**Pull request merged**